clients/bigquery/converters/converters_test.go

```diff
@@ -50,6 +49,12 @@ func TestStringConverter_Convert(t *testing.T) {
 			assert.NoError(t, err)
 			assert.Equal(t, "2021-01-01", val)
 		}
+		{
+			// Time
+			val, err := NewStringConverter(typing.Time).Convert(time.Date(2021, 1, 1, 9, 10, 11, 123_456_789, time.UTC))
+			assert.NoError(t, err)
+			assert.Equal(t, "09:10:11.123456Z", val)
```

Comment:
Is it okay that this has a different tz format than it did before? (`Z` instead of `+00`)

Reply:
Should be fine. This shouldn't actually impact anyone since the usage of "time" is rare and also this would require them to opt out of the "time" data type and move it to a "string"